### ISSUE TITLE
Comment form not displayed

### ISSUE THREAD

**Issue by kewwel** [getk2/k2-v3-dev-build#200]
Even with it enabled to appear, it does not appear. Just message: Login to post comments.
Need another setting for it to work as well as activate it in settings?

**Comment by kavadas** [comment]
Hi,

The front-end is not finalized yet. However i recommend to also check the permissions for comments.

**Comment by kewwel** [comment]
I checked and is enabled permissions. I grabbed a quickstart which is already enabled the comments and did the update and still continues to appear the message. I'll leave for testing, the stable version.

**Comment by kavadas** [comment]
May i ask, what kind of quick-start are you talking about?

**Comment by paulvanharen** [comment]
Hi Kavadas,

I can reproduce this issue. I've a clean install, and plugged k2 v3 straight into it. Then I created a category, an article and a test user. The K2 settings are configured to accept comments without captcha or whatever and publish these directly from registered users. 
In the article listing, the link "Be the first to comment!" shows up. Clicking that link opens up the article, but without a comments entry box. 

Next, I analyzed pieces of the source code.  In components/com_k2/views/item/view.html.php the scripts are loaded to support comment editing, but that's all. I.e. there is no other code instantiating an edit box. 

In components/com_k2/templates/default/item.php the content of the K2CommentsBlock is output to the HTML code. This block is created as the result of an onK2CommentsBlock event, but there is no plug in responding to the event. 

I hope the above helps nailing down the problem.

Best regards, Paul
